site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -58,6 +58,24 @@ You should see output similar to the following:
 {"message":"gameserver update received","severity":"info","time":"2019-10-30T21:46:18.179459+03:00"}
 ```
 
+{{% feature publishVersion="1.38.0" %}}
```

Comment:
This doesn't need to be feature gated - it's applicable everywhere., otherwise, this looks great!